src/Components/ConfirmDeleteDomain/ConfirmDeleteDomain.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ConfirmDeleteDomain from './ConfirmDeleteDomain';
+import '@testing-library/jest-dom';
+import { Domain } from '../../Api';
+
+const domain: Domain = {
+  domain_name: 'mydomain.test',
+} as unknown as Domain;
+
+test('expect empty when isOpen is false', () => {
+  const root = render(<ConfirmDeleteDomain domain={domain} isOpen={false} />);
+  expect(root.container).toBeEmptyDOMElement();
+});
+
+test('expect modal displayed', () => {
+  render(<ConfirmDeleteDomain domain={domain} isOpen={true} />);
+  expect(screen.getByRole('heading')).toHaveTextContent('Delete identity domain registration');
+  expect(screen.getAllByRole('button')[0]).toHaveTextContent('');
+  expect(screen.getAllByRole('button')[1]).toHaveTextContent('Delete');
+  expect(screen.getAllByRole('button')[2]).toHaveTextContent('Cancel');
+  async () => {
+    expect(await screen.findByText('Delete identity domain registration?')).toBe(
+      <>
+        Delete identity domain registration?
+        <br />
+        Now new host enrollment from HCC will be allowed on <b>{domain.domain_name}</b> domain after registration deletion.
```

Comment:
Should the displayed domain name be field: `domain.domain_name` or `domain.title` ?  I.e. with `domain_name` the displayed name is different with the value in column "name" in the domain list.

For this reason I've used `title` in https://github.com/podengo-project/idmsvc-frontend/pull/50/files#diff-487d23f05b405c8a325218f25d07497a626079b047abe5a3f4b4b832e2d172c5R49

We should agree on the same to use in both dialogs - to be consistent.

Reply:
Thanks! I have updated to use `domain.title` to be consistent :+1: